tests/testthat/test-pagination.R

```diff
@@ -316,3 +316,15 @@ test_that("non-monospaced fonts are caught", {
         )
     )
 })
+
+test_that("spans and string matrix match after pagination
+          when table has single column", {
+
+  df <- as.data.frame(mtcars[, 1])
+  test <- formatters:::basic_matrix_form(df)
+  pag_test <- formatters:::paginate_to_mpfs(test)
```

Comment:
```suggestion
  test <- basic_matrix_form(df)
  pag_test <- paginate_to_mpfs(test)
```
you do not need `:::` for internals in testthat